lib/ClimaCoreVTK/test/runtests.jl

```diff
@@ -88,6 +90,30 @@ mkpath(joinpath(OUTPUT_DIR, "series"))
         )
     end
     @test isfile(joinpath(OUTPUT_DIR, "series", "sphere_vector_series.pvd"))
+
+    writevtk(
+        joinpath(OUTPUT_DIR, "sphere_latlong_scalar_series"),
+        times,
+        (coords = coords,);
```

Comment:
`coords` here is a single field, not an array of fields

Reply:
That's right! Thanks for catching that. I fixed it in [0ddb2e9](https://github.com/CliMA/ClimaCore.jl/pull/567/commits/0ddb2e92006b71c7a4b44dae438f8187fc0c2658). Tests are passing now

Reply:
Rebased onto `main` and squashed